### PR TITLE
Fix/signer extension demo

### PR DESCRIPTION
**Pull request by DanielZhangReal**
test private transfer

---

Before we can merge this PR, please make sure that all the following items have been checked off:

- [ ] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/sdk/blob/main/CHANGELOG.md) and added the appropriate `L-` label to the PR.
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer.

### CODE REVIEW

File: manta-js/package/src/api/index.js [hunk 1]
```diff
@@ -6,35 +6,13 @@ import { base64Decode } from '@polkadot/util-crypto';
...
-//   const arrayBuffer = new ArrayBuffer(byteString.length);
-//   const _ia = new Uint8Array(arrayBuffer);
-//   for (let i = 0; i < byteString.length; i++) {
-//     _ia[i] = byteString.charCodeAt(i);
-//   }
-
-//   const dataView = new DataView(arrayBuffer);
-//   const blob = new Blob([dataView], { type: mimeString });
-//   return blob;
-// }
```
Comment (ghzlatarev) [inline manta-js/package/src/api/index.js:27]: is this commented code forgotten ?

File: manta-js/package/src/sdk.interfaces.ts [hunk 2]
```diff
@@ -48,16 +53,16 @@ export interface IMantaPrivateWallet {
...
   dropAuthorizationContext():any;
-  dropUserMnemonic():any;
+  dropUserSeedPhrase():any;
   convertPrivateAddressToJson(address: string): any
   getNetworks(): any;
   getZkAddress(): Promise<Address>;
   getAssetMetadata(assetId: BN): Promise<any>;
   initalWalletSync(): Promise<boolean>;
-  walletSync(): Promise<boolean>;
+  walletSync(forceSync?: boolean): Promise<boolean>;
```
Comment (ghzlatarev) [inline manta-js/package/src/sdk.interfaces.ts:65]: Can you add some description in the PR about the interface changes.

Comment (DanielZhangReal) [reply manta-js/package/src/sdk.interfaces.ts:65]: yeah, will update

File: manta-js/package/src/wallet/crate/Cargo.toml [hunk 3]
```diff
@@ -35,11 +35,11 @@ disable-restart = []
 console_error_panic_hook = { version = "0.1.7", default-features = false }
 getrandom = { version = "0.2.8", default-features = false, features = ["js"] }
 js-sys = { version = "0.3.60", default-features = false }
-manta-accounting = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.11", default-features = false, features = ["serde"] }
-manta-crypto = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.11", default-features = false, features = ["serde"] }
-manta-parameters = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.11", default-features = false }
-manta-pay = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.11", default-features = true, features = ["arkworks", "groth16", "http", "scale", "serde", "wallet"] }
-manta-util = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.11", default-features = false, features = ["serde", "reqwest"] }
+manta-accounting = { git = "https://github.com/manta-network/manta-rs.git", branch = "ledger_bug", default-features = false, features = ["serde"] }
```
[the remark this answers is not in the document]
Comment (SupremoUGH) [reply manta-js/package/src/wallet/crate/Cargo.toml]: It was a custom branch with panics to catch bugs. Now that they've all been caught I reverted to the release version.

File: manta-js/package/src/privateWallet.ts [hunk 4]
```diff
@@ -296,13 +301,12 @@ export class MantaPrivateWallet implements IMantaPrivateWallet {
   async toPrivateBuild(
     assetId: BN,
     amount: BN,
-    polkadotSigner: Signer,
```
Comment (ghzlatarev) [inline manta-js/package/src/privateWallet.ts:299]: nice

**Pull request closed without merging**